docs/quickstart-traffic-splitting.md

```diff
@@ -76,9 +76,16 @@ This `odh-model-controller` quick start demonstrates how to use the ModelMesh fr
 1. Enable ODH Service Mesh features in the `modelmesh-serving` namespace:
 
     ~~~
-    kubectl label namespace modelmesh-serving opendatahub.io/service-mesh=true
+    kubectl annotate namespace modelmesh-serving opendatahub.io/service-mesh=true
     ~~~
 
+1. Annotate the modelmesh-serving namespace to use the Istio Gateway:
+
+   ~~~
+   kubectl annotate namespace modelmesh-serving service-mesh.opendatahub.io/public-gateway-name=opendatahub/odh-gateway
+   kubectl annotate namespace modelmesh-serving service-mesh.opendatahub.io/public-gateway-host-internal=istio-ingressgateway.istio-system.svc.cluster.local
```

Comment:
Similar comment here.
Was this example experimental? Should we think at a migration procedure?

Reply:
As this is going to a `service-mesh-integration` feature branch, we are telling this is in "incubation". So, yes, API is subject to change until all this is merged back to `main`.

Reply:
Thanks for the confirmation :)